platform/android/MapboxGLAndroidSDKTestApp/src/main/java/org/maplibre/android/testapp/activity/annotation/PolygonActivity.kt

```diff
@@ -56,18 +55,16 @@ class PolygonActivity : AppCompatActivity(), OnMapReadyCallback {
     override fun onMapReady(map: MapLibreMap) {
         maplibreMap = map
         map.setStyle(Style.getPredefinedStyle("Streets"))
-        map.setOnPolygonClickListener { polygon: Polygon ->
+        /*map.setOnPolygonClickListener { polygon: Fill ->
             Toast.makeText(
                 this@PolygonActivity,
                 "You clicked on polygon with id = " + polygon.id,
```

Comment:
Can this example be revived?

Reply:
@louwers

> `map.setOnPolygonClickListener` still works right?

Actually, no, this is not covered by the spec. Probably it should still be added – I think it would be best to do this at the same time that we add functionality for users being able to add managers to `MapLibreMap` (which, now to notice, is not actually covered by any of the existing issues. I'd suggest to do it after automatic grouping is implemented since we wanted a method that adds a new manager above a specific z layer).

I reinstated the click listener by adding it to the individual polygon.